src/platform_impl/windows/event_loop.rs

```diff
@@ -1200,7 +1200,7 @@ unsafe fn public_window_callback_inner<T: 'static>(
 
             let value = (wparam >> 16) as i16;
             let value = value as i32;
-            let value = value as f32 / winuser::WHEEL_DELTA as f32;
+            let value = -value as f32 / winuser::WHEEL_DELTA as f32;
```

Comment:
https://docs.microsoft.com/en-us/windows/win32/api/windowsx/nf-windowsx-get_y_lparam

Reply:
@aloucks could you expand on the relevance of this please? I'm not familiar with the win32 api.

Reply:
Is the suggestion to use `lparam` instead of `wparam`?

Reply:
I think I misread part of https://docs.microsoft.com/en-us/windows/win32/inputdev/wm-mousehwheel and thought the negative value might be related. I think you can disregard my previous comment.

Have you checked to see how GLFW treats the mouse wheel direction? I think we should just use that prior art as reference and have the same behavior.

Reply:
GLFW reference: https://github.com/glfw/glfw/blob/53d86c64d709ff52886580d338d9b3b2b1f27266/src/win32_window.c#L935-L941

Reply:
> GLFW reference: https://github.com/glfw/glfw/blob/53d86c64d709ff52886580d338d9b3b2b1f27266/src/win32_window.c#L935-L941

Awesome, if I read this correctly, than this PR follows GLFW prior art.